src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,63 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manSer =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService drSer =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final CarService carSer =
```

Comment:
abbreviations mostly never provide benefit but usually lead to problems with code reading
```suggestion
    private static final ManufacturerService manufacturerService =
            (ManufacturerService) injector.getInstance(ManufacturerService.class);
    private static final DriverService driverService =
            (DriverService) injector.getInstance(DriverService.class);
    private static final CarService carSer =
```